rustup-macros/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+edition = "2021"
+name = "rustup-macros"
+publish = false
+version = "0.1.0"
```

Comment:
```suggestion
name = "rustup-macros"
publish = false
version = "0.1.0"
edition = "2021"
```

Reply:
Why this order? I have a IDE plugin that sorts toml keys, which reduces merge conflicts (because folk aren't all adding at the end of the same list). You can see that this is sorted alphabetically.

Reply:
Because Cargo always keeps the name first when you create a new package.

Reply:
Maybe alphabetic order is more make sense.